### PR TITLE
HAAR-1194: 🐛 maintain group page - search for users link updated

### PR DESCRIPTION
**Pull request by SimonMitchellMOJ**

### CODE REVIEW

File: views/groupDetails.njk [hunk 1]
```diff
@@ -163,7 +163,8 @@
     }) }}
     {% endif %}
 
-    {% set searchUrl = '/search-external-users/results?groupCode='+ groupDetails.groupCode %}
+    {% set searchUrl = '/search-external-users?user=&status=ALL&roleCode=&groupCode='+ groupDetails.groupCode %}
+
```
Comment (karenmillermoj) [inline views/groupDetails.njk:167]: I'd be tempted to just pass through the groupCode.  It should just pick up the others as default

File: integration-tests/e2e/groups.cy.js [hunk 2]
```diff
@@ -74,6 +76,30 @@ context('Groups', () => {
...
+    cy.task('stubExternalUserSearch', {
+      content: replicateUser(1),
+      totalElements: 1,
+      page: 0,
+      size: 1,
+    })
+    groupDetails.searchUsers()
+
+    ExternalUserSearchPage.verifyOnPage()
+  })
```
Comment (karenmillermoj) [inline integration-tests/e2e/groups.cy.js:101]: Do we need to check if the page is filtering by group code?

Comment (SimonMitchellMOJ) [reply integration-tests/e2e/groups.cy.js:101]: that is cover by the normal search

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**